_data/projects/NbaSearch.yml

```diff
@@ -0,0 +1,17 @@
+name: NBA Search
+desc: An NBA analytics web application which contains a chatbot, blogs, and predictions. Great project for early data scientists and machine learning engineers. 
+site: https://github.com/skekre98/NBA-Search
+tags:
+  - python
+  - javascript
+  - html
+  - css
+  - flask
+  - pandas
+  - sklearn
+  - bs4
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/skekre98/NBA-Search/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
+  name: good first issue
```

Comment:
Currently we don't support multiple listings in the single file - #244 is our tracking issue for that.

I'd recommend choosing one label to list here to keep it simple - up to you which one you'd prefer to see.

Reply:
@shiftkey removed the extra tag. Let me know if there is anything else that needs updating 👍